mailserver/cleaner_test.go

```diff
@@ -82,14 +84,15 @@ func archiveEnvelope(t *testing.T, sentTime time.Time, server *WMailServer) *whi
 func testPrune(t *testing.T, u time.Time, expected int, c *Cleaner, s *WMailServer) {
 	upper := uint32(u.Unix())
 	_, err := c.Prune(0, upper)
-	assert(err == nil, "", t)
+	require.NoError(t, err)
+
 	count := countMessages(t, s.db)
-	assert(count == expected, fmt.Sprintf("expected %d message, got: %d", expected, count), t)
+	require.Equal(t, expected, count, fmt.Sprintf("expected %d message, got: %d", expected, count))
```

Comment:
Nitpick: we can also remove the message as almost the same will be printed by `testify`.